### PR TITLE
MicroFocus SMAX Integration

### PR DESCRIPTION
**Pull request by ayman-m**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Fetch SMAX incidents , requests and automate different SMAX case management actions

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ x] No

## Must have
- [ ] Tests
- [ x] Documentation

### CODE REVIEW

**Comment by iyeshaya** [comment]
Hi @ayman-m , we appreciate your effort on contributing to the XSOAR marketplace. Due to excessive amount of contributions in our queue, it may take us a bit more time than usual to review this pull request, we appreciate your patience.

In the meanwhile:
- Please go over the failed [validations](https://app.circleci.com/pipelines/github/demisto/content/123508/workflows/50e27894-d426-4002-bf4b-3a9d679e1ba3/jobs/440816) + [Lint](https://app.circleci.com/pipelines/github/demisto/content/123508/workflows/50e27894-d426-4002-bf4b-3a9d679e1ba3/jobs/440817) and make the necessary fixes.
Feel free to comment here any question that may raise.

File: Packs/MicroFocusSMAX/Integrations/MicroFocusSMAX/MicroFocusSMAX.yml [hunk 1]
```diff
@@ -0,0 +1,545 @@
...
+- defaultvalue: Incident
+  display: Object To Fetch
+  hidden: false
+  name: object_to_fetch
+  options:
+  - Incident
+  - Request
+  required: false
+  type: 15
+- defaultvalue: https://us1-smax.saas.microfocus.com
```
Comment (iyeshaya) [inline Packs/MicroFocusSMAX/Integrations/MicroFocusSMAX/MicroFocusSMAX.yml:15]: Hey @ayman-m, it seems like you are failing on the secrets validations.
You can either remove the secret or add it to the secrets-ignore file.
For more information read here https://xsoar.pan.dev/docs/concepts/demisto-sdk#secrets

**Review by iyeshaya — CHANGES_REQUESTED** [review]

File: Packs/MicroFocusSMAX/Integrations/MicroFocusSMAX/MicroFocusSMAX.yml (hunk 1 above)
Comment (ayman-m) [reply Packs/MicroFocusSMAX/Integrations/MicroFocusSMAX/MicroFocusSMAX.yml:15]: Hey @iyeshaya , I think this is just a false positive ,  https://us1-smax.saas.microfocus.com is not a secret, it's Microfocus cloud console url

Comment (iyeshaya) [reply Packs/MicroFocusSMAX/Integrations/MicroFocusSMAX/MicroFocusSMAX.yml:15]: That's exactly why we have the secret-ignore file, everything inside this file will be ignored during the secret validation process.
Please add ```https://us1-smax.saas.microfocus.com``` to this file.
Thank you.

Comment (ayman-m) [reply Packs/MicroFocusSMAX/Integrations/MicroFocusSMAX/MicroFocusSMAX.yml:15]: I have added it to .secrets-ignore

**Review by iyeshaya — APPROVED** [review]

**Pull request merged**